docs/admin/install.rst

```diff
@@ -207,14 +209,14 @@ In order to decrypt submissions, your SecureDrop Workstation will need a copy of
   .. code-block:: sh
 
     qvm-run --pass-io vault \
-      "gpg --homedir /run/media/user/TailsData/gnupg -K --fingerprint"
```

Comment:
I plugged my Admin Workstation drive into a Fedora 37 VM and it mounted in `/run/media/...` (the old paths). Do I need to do something different? When did the paths change?